src/test/java/lotto/domain/LottosTest.java

```diff
@@ -2,19 +2,27 @@
 
 import static org.assertj.core.api.Assertions.*;
 
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-public class LottosTest {
-	@DisplayName("랜덤 번호가 1~45의 숫자인지 확인.")
+public final class LottosTest {
+	@DisplayName("당첨 번호를 포함하는지 비교해서 포함하는 개수를 반환한다.")
 	@Test
-	void 랜덤_번호가_1부터45까지의_숫자인지_확인() {
-		Lottos lottos = new Lottos();
-		List<Lotto> shuffledLottos = lottos.allLottoNumbers().shuffle();
+	void 당첨번호를_포함하는지_비교해서_개수를_반환한다() {
+		int[] winningNumbers = {1, 2, 3, 4, 5, 7};
+		List<Lotto> lottoList = Arrays.asList(new Lotto(1), new Lotto(2),
+			new Lotto(3), new Lotto(4), new Lotto(5), new Lotto(6));
```

Comment:
```suggestion
		Lottos lottos = new Lottos(1, 2, 3, 4, 5, 6);
```
생성자를 추가해 위와 같이 구현 가능해본다.
힌트는 가변 인자